README.md

```diff
@@ -1,5 +1,20 @@
 # Regulatory R Package Repository
 
+Using R in submissions to healthcare regulators is challenging since the 
+quality of any packages used must be ensured and evidence of ensuring this
+quality must be made available on request. 
+This working group strives to identify and prototype at least one technical
+framework that can support a transparent, open, dynamic, cross-industry approach
+of establishing and maintaining a 'repository' of core R packages with
```

Comment:
Just stepping back the "core" qualifier, since I think that scope is still TBD

```suggestion
of establishing and maintaining a 'repository' of R packages with
```